src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -34,6 +34,9 @@ def filterDataByTier(rawData, allowedTiers):
     :return: the same dictionary as in the input, but without dataset structs
              for datatiers that we do not want to be processed by this component.
     """
+    if not allowedTiers:
```

Comment:
I do not like this magical silent metamorphosis of the value name from `listTiersToInject` to `allowedTiers`. Those are linked only on line 171 in the current commit: [1]. This makes the code hard to track. I would suggest this value to remain consistent with it's configuration name [2] wherever it is passed to a function so that it's meaning remains unambiguous throughout the code.

[1]
https://github.com/dmwm/WMCore/blob/a3624ea191c980555b0464aae68866c1bf231c35/src/python/WMComponent/RucioInjector/RucioInjectorPoller.py#L171

[2]
https://github.com/dmwm/WMCore/blob/a3624ea191c980555b0464aae68866c1bf231c35/src/python/WMComponent/RucioInjector/RucioInjectorPoller.py#L88

Reply:
Honestly, I do not know what's the most pythonic way to pass these variables. I know that pylint complains if we pass a variable named `blah` to a function with a parameter named the same (thus `blah`).

It can be easier to track indeed, I'll update it.

Reply:
Thanks